scripts/dependency-generator.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-sync */
+const fs = require('fs');
+const path = require('path');
+const globby = require('globby');
+const file = 'release-dependencies.md';
+
+fs.writeFileSync(file, '', 'utf-8');
+
+const updateFile = (content = '') => {
+    fs.appendFileSync(file, `${content}\n`, 'utf-8');
+};
+
+const packagesFiles = globby.sync([
+    '**/*/package.json',
+    '!**/*/fixtures'
+], { gitignore: true });
+
+const packages = new Map();
+const dependsOn = new Map();
+
+packagesFiles.forEach((pkgFile) => {
+    const destination = path.join(process.cwd(), pkgFile);
+    const pkg = require(destination);
+
+    packages.set(pkg.name, pkg);
+
+    const deps = [
+        ...Object.keys(pkg.dependencies || {}),
+        ...Object.keys(pkg.devDependencies || {}),
+        ...Object.keys(pkg.peerDependencies || {}),
+        ...Object.keys(pkg.optionalDependencies || {})
+    ].filter((name) => {
+        return name.startsWith('@hint/');
```

Comment:
what about `hint`? don't you want to add that dependency?

Reply:
You are right, but you can ignore this script as it was only for testing purposes (although it might get into the new release process in some form)